sanitizers/detectors/detector.go

```diff
@@ -1,9 +1,103 @@
 package detectors
 
-// Detector defines the interface for the bug detectors.
+import (
+	"os/exec"
+	"text/template/parse"
+)
+
+// Define an enum variant, for each implemented `Detector`
+type Detectors int64
+
+const (
+	SQLInjection Detectors = iota
+	TemplateInjection
+	PathTraversal
+	CommandInjection
+)
+
+var SQLIDet Detectors = SQLInjection
+var TIDet Detectors = TemplateInjection
+var PTDet Detectors = PathTraversal
+var CIDet Detectors = CommandInjection
```

Comment:
I think abbreviated names like these makes it rather difficult to directly know what the variables represent in the code.